res/values/arrays.xml

```diff
@@ -2,76 +2,76 @@
 <resources>
   <string-array name="pref_layout_values">
     <item>none</item>
-    <item>azerty</item>
-    <item>qwerty</item>
-    <item>qwerty_pt</item>
-    <item>qwerty_es</item>
-    <item>qwerty_el</item>
-    <item>qwerty_ko</item>
-    <item>qwerty_lv</item>
-    <item>qwerty_no</item>
-    <item>qwerty_sv_se</item>
-    <item>qwerty_hu</item>
-    <item>qwerty_tr</item>
-    <item>qwerty_pl</item>
-    <item>qwerty_vi</item>
-    <item>ru_jcuken</item>
-    <item>qwertz</item>
-    <item>qwertz_cs</item>
-    <item>qwertz_de</item>
-    <item>qwertz_hu</item>
-    <item>qwertz_sk</item>
-    <item>bgph1</item>
-    <item>dvorak</item>
-    <item>colemak</item>
-    <item>neo2</item>
-    <item>bone</item>
-    <item>jcuken_ua</item>
-    <item>bangla</item>
-    <item>devanagari_1</item>
-    <item>devanagari_2</item>
-    <item>he_il_1452_1</item>
-    <item>he_il_1452_2</item>
-    <item>ar_pc</item>
-    <item>ar_alt</item>
-    <item>persian</item>
+    <item>latn_qwerty_us</item>
+    <item>latn_colemak</item>
+    <item>latn_dvorak</item>
+    <item>arab_ibm</item>
+    <item>arab_ibm_ma</item>
+    <item>arab_ir</item>
+    <item>beng_rh</item>
+    <item>cyrl_jcuken_ru</item>
+    <item>cyrl_jcuken_uk</item>
+    <item>cyrl_yaverti</item>
+    <item>deva_inscript</item>
+    <item>deva_lv</item>
+    <item>grek_qwerty</item>
+    <item>hang_dubeolsik_kr</item>
+    <item>hebr_</item>
+    <item>hebr_2</item>
+    <item>latn_azerty_fr</item>
+    <item>latn_bone</item>
+    <item>latn_neo2</item>
+    <item>latn_qwerty</item>
+    <item>latn_qwerty_br</item>
+    <item>latn_qwerty_es</item>
+    <item>latn_qwerty_lv</item>
+    <item>latn_qwerty_no</item>
+    <item>latn_qwerty_pl</item>
+    <item>latn_qwerty_se</item>
+    <item>latn_qwerty_tr</item>
+    <item>latn_qwerty_vi</item>
+    <item>latn_qwertz</item>
+    <item>latn_qwertz_cz</item>
+    <item>latn_qwertz_de</item>
+    <item>latn_qwertz_hu</item>
+    <item>latn_qwertz_sk</item>
     <item>custom</item>
   </string-array>
   <string-array name="pref_layout_entries">
     <item>None</item>
-    <item>AZERTY</item>
-    <item>QWERTY</item>
-    <item>QWERTY (Brasileiro)</item>
-    <item>QWERTY (Español)</item>
-    <item>QWERTY (Greek)</item>
-    <item>QWERTY (Korean)</item>
-    <item>QWERTY (Latvian)</item>
-    <item>QWERTY (Norwegian)</item>
-    <item>QWERTY (Swedish)</item>
-    <item>QWERTY (Hungarian)</item>
-    <item>QWERTY (Türkçe)</item>
-    <item>QWERTY (Polski)</item>
-    <item>QWERTY (Vietnamese)</item>
-    <item>ЙЦУКЕН (Русский)</item>
-    <item>QWERTZ</item>
-    <item>QWERTZ (Czech)</item>
-    <item>QWERTZ (Deutsch)</item>
-    <item>QWERTZ (Hungarian)</item>
-    <item>QWERTZ (Slovak)</item>
-    <item>Bulgarian (Phonetic Traditional)</item>
-    <item>Dvorak</item>
+    <item>QWERTY [United States of America]</item>
```

Comment:
This is more commonly called "US". Shorter is better.

Reply:
Or “USA”, because strictly speaking, “United States” can also indicate Mexico as it is called “United Mexican States”.

Reply:
"US" is the most used. Let's keep it simple.